app/components/post_list/post_list.tsx

```diff
@@ -123,21 +125,25 @@ const PostList = ({
         return orderedPosts.findIndex((i) => i.type === 'start-of-new-messages');
     }, [orderedPosts]);
 
-    const isNewMessage = lastPostId ? posts[0]?.id !== lastPostId : false;
+    const isNewMessage = lastPostId ? firstIdInPosts !== lastPostId : false;
+
+    const scrollToEnd = useCallback(() => {
+        listRef.current?.scrollToOffset({offset: 0, animated: true});
+    }, []);
```

Comment:
This was moved from below and renamed from 'onScrollToEnd' to 'scrollToEnd' so that it can be used multiple times (136, 146 and of course 363).